packages/roosterjs-editor-plugins/lib/plugins/ImageEdit/ImageEdit.ts

```diff
@@ -504,6 +506,7 @@ export default class ImageEdit implements EditorPlugin {
                     rotateHandle
                 );
 
+                checkIftheImageWasResized(this.image, context);
```

Comment:
Actually, when run into line 489 (or line 487 before your change), it means user has dragged resize handle and we can treat it as image resized.

You can add some new optional property to ImageEditInfo to save this resized state. Then only set image width/height when this value is true, or remove it when this is false. So no need to have additional "wasReset" or "wasResize" attribute

But if the image already has width/height initially, always set this value to true.

Reply:
I added this attribute in case of the image was deleted, so if the image is deleted, the attribute is deleted too, then I don't have to reset a property in case of the image is deleted.  Also, I'm not using the image size to check if it was resizes, because if the user changes the screen size, it will change the size of the image without resize plugin be called.

Reply:
I'm not suggesting comparing image size. Instead, I mean we check if the resize handle is dragged by user, that will result in line 489 to be called.

Here's the flow:
- User drag  resize handle => 
- Resize.onDrag() is called and return true => 
- DragAndDropHelper call onSubmit() => 
- ImageEditPlugin.updateWrapper() is called with parameter "context" => 
- Line 489 is triggered

So that it is best place to remember that user has really "resized" the image. If user change screen size, since user didn't drag the resize handle, those code won't be triggered.

What is your concern of image is deleted?

Reply:
I changed the code to check if the image is resized after line 489. 
My concern is to use a property to store if the image was resized and even if delete the image, this property keep stores true as value.

Reply:
If you store the value in ImageEditInfo, it will be serialized to the image. So when image is deleted, the serialized result is deleted as well.